test/dicer-multipart-extra-trailer.spec.js

```diff
@@ -1,9 +1,10 @@
 const Dicer = require('../deps/dicer/lib/Dicer');
 const assert = require('assert'),
   fs = require('fs'),
+  path = require('path'),
   inspect = require('util').inspect;
 
-const FIXTURES_ROOT = __dirname + '/fixtures/';
+const FIXTURES_ROOT = path.join(__dirname, '/fixtures/');
```

Comment:
slash before fixture is not needed with join

Reply:
done